Panto/css/general.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+  line-height: var(--line-height-body);
+  font-weight: var(--font-weight-regular);
+  color: var(--color-body);
+  overflow-x: hidden;
+}
+
+html {
+  font-size: var(--font-size-html);
+}
+
+/* General Components */
+.container {
+  /* 1440px */
+  max-width: 128rem;
+  padding-inline: 8rem;
+  margin-inline: auto;
+}
+
+h1,
+h2,
+h3 {
+  font-weight: var(--font-weight-bold);
+}
+
+h1 {
+  font-size: var(--font-size-h1);
+  line-height: var(--line-height-h1);
+}
+
+h2 {
+  font-size: var(--font-size-h2);
+  line-height: var(--line-height-h2);
+}
+
+h3 {
+  font-size: var(--font-size-h3);
+  line-height: var(--line-height-h3);
+}
+
+.subheading {
+  display: block;
+  font-size: var(--font-size-subheading);
+  font-weight: var(--font-weight-medium);
+  text-transform: uppercase;
+  margin-bottom: 2rem;
```

Comment:
to make everything feel consistent, it is better to use the `margin-block-end` property